src/GeneralStateTestsFiller/stCreateTest/CREATE_EOF1Filler.yml

```diff
@@ -0,0 +1,174 @@
+# CREATE opcode deploying code with 0xef first byte (EIP-3541)
+CREATE_EOF1:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 1000000
+      code: ''
+      nonce: 0
+      storage: {}
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      code: ':yul { mstore(0, calldataload(0)) sstore(0, create(0, 0, calldatasize())) }'
+      nonce: 0
+      storage: {}
+
+  transaction:
+    data: 
+      #- ':yul { mstore8(0, 0xef) return(0, 1) }'
+      - ':label deploying_0xef :raw 0x60ef60005360016000f3'
+      #- ':yul { mstore8(0, 0xef) return(0, 2) }'
+      - ':label deploying_0xef00 :raw 0x60ef60005360026000f3'
+      #- ':yul { mstore8(0, 0xef) return(0, 3) }'
+      - ':label deploying_0xef0000 :raw 0x60ef60005360036000f3'
+      #- ':yul { mstore8(0, 0xef) return(0, 32) }'
+      - ':label deploying_0xef00..00 :raw 0x60ef60005360206000f3'
+    gasLimit:
+      - 700000
+    gasPrice: 1
+    nonce: 0
+    secretKey: 45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8
+    to: 'b94f5374fce5edbc8e2a8697c15331677e6ebf0b'
+    value:
+    - 0
+
+  expect:
+    - indexes:
+       data: ':label deploying_0xef'
+       gas: !!int -1
+       value: !!int -1
+      network:
+        - 'Berlin'
```

Comment:
Would this work?

```suggestion
        - '<=Berlin'
```

Reply:
Nonce for created accounts was 0 before state-clearing fork, so we would need either make tests a bit more complicated with another nonce expectation for old forks, or remove nonce expectation, or list only forks starting from SpuriousDragon.

But it looks usual in new tests to check only new behaviour and maximum of one previous fork, not caring about repeating it for old forks with the same rules. @winsvega can correct me if it's not ok approach.

Reply:
Yes because we have copy of the tests in legacy with old forks expect section. However new tests are targeting only recent forks (as also pointed out by Alexy Akhunov)

Since we already have client consensus with existing tests, no really need to cover previous forks.

You can use a sstore operation and check that it worked (instead of nonce)

Another issue when having tests with 5 or more forks it takes more time to generate and run it. So I plan to archive tests from time to time refreshing the expect section to the most recent fork only.

Reply:
for full node implementation IMHO it is good to have tests that check that new changes does not break anything in the past. so if you really think we should check it here we can make an exception.